control/content/index.html

```diff
@@ -3,263 +3,26 @@
 <head lang="en">
     <meta charset="UTF-8">
 
-	<!-- CSS -->
+    <!-- CSS -->
     <link href="../../../../styles/helper.css" rel="stylesheet">
+    <link href="content.app.css" rel="stylesheet">
 
-	<!-- JS -->
+    <!-- JS -->
+    <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
```

Comment:
Hi Daniel, I guess there were two different pull request created. That might be the reason of the issue.